policyengine_us/variables/gov/states/id/tax/income/deductions/id_aged_or_disabled_deduction.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_us.model_api import *
+
+
+class id_aged_or_disabled_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho aged or disabled deduction"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
```

Comment:
The law/form look to refer to household members even if they're not dependents in the tax unit. If this is the case, we may want `tax_unit.household.members`, though I'm not immediately sure how to filter out heads/spouses of the specific tax unit (as opposed to other tax units in the household). If I misrepresented please change it.
```suggestion
        # The program includes individuals in the household, including those not in the filing unit.
        # We simplify by limiting to those in the filing unit.
        person = tax_unit.members
```